src/client/components/PullRequestList/PullRequestListItem/index.js

```diff
@@ -65,10 +65,10 @@ class PullRequestListItem extends Component {
                   style={{ textDecoration: 'none', color: 'rgb(59, 120, 155)' }}
                   to={{
                     pathname: buildProjectLink(pullRequest.origin.repository.fullName),
-                    query: { branch: pullRequest.origin.branch } }}
+                    query: { branch: pullRequest.origin.name } }}
```

Comment:
This link is only valid if the `origin.type === 'BRANCH'`. Should we handle REV, TAG and the other types here?